hibernate-core/src/main/java/org/hibernate/query/spi/QueryParameterBindings.java

```diff
@@ -35,4 +35,12 @@
 	Type[] collectPositionalBindTypes();
 	Object[] collectPositionalBindValues();
 	Map<String,TypedValue> collectNamedParameterBindings();
+
+	/**
+	 * @deprecated expect a different approach to org.hibernate.engine.spi.QueryParameters in 6.0
+	 */
+	@Deprecated
+	default boolean isAParameterList(QueryParameter parameter) {
```

Comment:
The `A` in the name is irrelevant.  The name should be `#isParameterList`.  But personally I like `#isMultiValuedBinding`  better

Reply:
I like `isMultiValuedBinding`, going to change it. Thanks